### ISSUE TITLE
Error loading debug manifests

### ISSUE THREAD

**Issue by vaderj** [SharePoint/sp-dev-docs#4555]
===&gt;   https://imgur.com/a/hm6coXK   &lt;===  Image of the error 

Step 10 of the "Debug your Field Customizer" section fails - after "gulp serve", the page loads, requests permission to execute the code, and immediately after allowing the code execution, this error occurs.

Searching through all files in the project for "debugManifestsFile" returns 0 results.  Where should this be defined?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cbc14882-1021-df0e-73c1-95d049f091d7
* Version Independent ID: eda24750-4e60-f05c-a6bb-0cc017dfe9ed
* Content: [Build your first Field Customizer extension](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/building-simple-field-customizer)
* Content Source: [docs/spfx/extensions/get-started/building-simple-field-customizer.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/spfx/extensions/get-started/building-simple-field-customizer.md)
* Product: **sharepoint**
* Technology: **sharepoint-framework**
* GitHub Login: @spdevdocs
* Microsoft Alias: **spdevdocs**

**Comment by andrewconnell** [comment]
Need additional context, such as SharePoint environment/version, SPFx version, browsers tested, etc. More context makes it easier to understand the question & help...

**Comment by vaderj** [comment]
Doh!

SharePoint Online
@microsoft/generator-sharepoint	1.8.1
Chrome 76.0.3809.132 / IE11.678 has the same behavior
Node v10.15.3
Win10 1809

**Comment by andrewconnell** [comment]
You won't find `debugManifestsFile`... it's not a file, that's a property pointing to **/temp/manifests.js** on your local machine and served up by the webserver started by the **gulp serve** command.

Couple questions:

1. Does your local workbench load and can you interact with it (you won't see your extension, but generally, does it "work")?
1. What is the _FULL_ URL that was used to open the SPO page for debugging
1. When you hit that page, look at the console... do you see (usually pink/purple) logs that report files requested and served up? This usually shows up right after the stuff you see in the screenshot before step 10.

> _BTW, you can paste images into the comments... don't have to save them to Imgur._

**Comment by vaderj** [comment]
1.) I don't know if the workbench is loading.  If I close the error modal, I can interact with my SharePoint list as normal: https://imgur.com/4RvGF7j


2.)  https://vader.sharepoint.com/Lists/test/AllItems.aspx?debugManifestsFile=https%3A%2F%2Flocalhost%3A4321%2Ftemp%2Fmanifests.js&loadSPFX=true&fieldCustomizers=%7B%22InternalFieldName%22%3A%7B%22id%22%3A%222e66ef3e-41e5-454e-80a8-bad02ead3ea8%22%2C%22properties%22%3A%7B%22sampleText%22%3A%22Value%22%7D%7D%7D

3.) Gulp output: https://imgur.com/iephS5H

I do not see any activity from Gulp when I reload the page in my browser.


I just now saw your comment about pasting images ....

**Comment by andrewconnell** [comment]
There's not a single call back to your localhost in that last screenshot, which is why you are seeing that error. You're leaving the console up when you browse to the page, correct? 

In the screenshot the local webserver is terminating ~30s after it started... are you doing that before you press the dialog that prompts to load debug scripts?

This is what you should see:
![image](https://user-images.githubusercontent.com/2068657/63969411-d1b52100-ca6f-11e9-940e-3e0b0f8b0ce9.png)

Can you hit your localhost when developing a web part?

**Comment by vaderj** [comment]
The local server was terminated by me, it runs "fine" otherwise.

When I hit localhost endpoint, Chrome tells me the cert is invalid though I have ran

gulp trust-dev-cert

multiple times now (in elevated prompt) and its shown no errors
![image](https://user-images.githubusercontent.com/5408195/63970531-43ce3b80-ca59-11e9-9f05-27feda40cb00.png)


I am guessing that the cert error is the problem

**Comment by andrewconnell** [comment]
Yup... that's absolutely the first thing you need to address. 90% sure that's the problem.

There are lots of open/closed issues on this topic in this list. Please review those for some solutions and come back if you are still having issues. Also look at the docs for help: https://docs.microsoft.com/en-us/sharepoint/dev/spfx/set-up-your-development-environment#unable-to-trust-the-self-signed-development-certificate

Quick checklist:
- untrust it, reboot, launch admin command prompt & re-trust it
- delete that hidden `gcb*` folder
